### ISSUE TITLE
Logger's documentation is hard to find

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7904]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#7904|
|Type|defect|
|Created|2015-05-19 15:20:45Z|
|Branch|https://github.com/twisted/twisted/tree/log-docindex-7904|

It's on the ToC in http://twistedmatrix.com/documents/current/core/howto/index.html , but doesn't have a second link and description like all the others. http://twistedmatrix.com/documents/current/core/howto/logging.html#note-new-logging-system should also link to it directly.

<details><summary>Searchable metadata</summary>

```
trac-id__7904 7904
type__defect defect
reporter__hawkowl hawkowl
priority__normal normal
milestone__New_Logging_System New Logging System
branch__branches_log_docindex_7904 branches/log-docindex-7904
branch_author__wsanchez wsanchez
status__closed closed
resolution__fixed fixed
component__logger logger
keywords__ 
time__1432048845198007 1432048845198007
changetime__1481165896808986 1481165896808986
version__None None
owner__hawkowl hawkowl
cc__z3p cc__hynek
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)|@wsanchez set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_5"></a>|@wsanchez commented|
|-|-|

(In [44871]) Branch for [#7904](https://github.com/twisted/twisted/issues/7904)

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_6"></a>|@wsanchez commented|
|-|-|

Grr.  Tried what looked to be the obvious things and that didn't work…

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_7"></a>|@glyph commented|
|-|-|

Since there were no actual commits on this branch, I took the liberty of trying to fix it myself :).

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_8"></a>|@glyph removed owner<br>@glyph set status to `new`|
|-|-|

Builders are running now.

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)|@wsanchez set owner to @wsanchez<br>@wsanchez set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_10"></a>|@wsanchez commented|
|-|-|

reviewed.

Note that in the old HOWTO, the text "its API reference here" is in monospace, which is odd because that's not code.  Not sure that's avoidable due to how `:api:` works.

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_11"></a>|@wsanchez commented|
|-|-|

(please land)

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_12"></a>|@glyph commented|
|-|-|

Yeah we should probably do a rev on `:api:` at some point, since there's no way to link and say "this is prose".

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph set owner to @hawkowl<br>@glyph set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_14"></a>|@glyph commented|
|-|-|

Since HawkOwl said she wanted to do something weird for the 15.2.1 release, I'm going to leave this open:

```
[09:12:06] <hawkowl> please don't merge #7903/#7904, i think i'll have to merge them into a 15.2.1 branch and then merge that into trunk
```

But hawkie, for future reference, please add some kind of workflow tag to tickets like this one, and maybe document on the wiki what needs to happen.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_15"></a>|@hawkowl set status to `closed`|
|-|-|

(In [44880]) Merge log-docindex-7904: Update links to logger's documentation.

Author: glyph
Reviewer: wsanchez
Fixes: [#7904](https://github.com/twisted/twisted/issues/7904)
